### DISCUSSION TITLE
speed decoding / displaying images

### DISCUSSION THREAD

**Discussion by stef-ladefense** [Bodmer/TFT_eSPI#3523] [Q&A - General]
I have several small images of about 117x77 stored in the internal flash in a littlesfs partition.
use ESP32S3 N16R8

what do you think would be the format that would take the least time to decode / decompress? bmp? jpg? png? others?
currently I use bmp with the routine from the examples here to display them.

thanks

**Comment by Basitadam** [comment]
You can always decode them before the compile into .c or .h files. But if the images are updated into file system after the compile time, .bmp or .tiff should be the fastest since there is no compression on them
